src/ucp/core/ucp_ep.c

```diff
@@ -2429,11 +2429,11 @@ ucs_status_t ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config,
     config->tag.lane                    = UCP_NULL_LANE;
     config->tag.proto                   = &ucp_tag_eager_proto;
     config->tag.sync_proto              = &ucp_tag_eager_sync_proto;
-    config->tag.rndv.rma_thresh.remote  = SIZE_MAX;
-    config->tag.rndv.rma_thresh.local   = SIZE_MAX;
-    config->tag.rndv.am_thresh          = config->tag.rndv.rma_thresh;
-    config->rndv.rma_thresh             = config->tag.rndv.rma_thresh;
-    config->rndv.am_thresh              = config->tag.rndv.am_thresh;
+    config->tag.rndv_thresh.rma.remote  = SIZE_MAX;
+    config->tag.rndv_thresh.rma.local   = SIZE_MAX;
+    config->tag.rndv_thresh.am          = config->tag.rndv_thresh.rma;
+    config->rndv.thresh.rma             = config->tag.rndv_thresh.rma;
+    config->rndv.thresh.am              = config->tag.rndv_thresh.am;
```

Comment:
this is protov1 code, can we avoid changing it?

Reply:
Reverted related changes.